scripts/cm_test_all_sandia

```diff
@@ -319,6 +340,14 @@ if [ "$MACHINE" = "sems" ]; then
                "clang/3.9.0 $BASE_MODULE_LIST "Pthread_Serial" clang++ $CLANG_WARNING_FLAGS"
                "cuda/9.2 $CUDA9_MODULE_LIST $CUDA_BUILD_LIST $KOKKOS_PATH/bin/nvcc_wrapper $CUDA_WARNING_FLAGS"
     )
+  elif [ "$SPOT_CHECK_TPLS" = "True" ]; then
```

Comment:
Is this a separate branch in case the TPL spot check build changes in the future? It looks like the set of builds is identical to the normal spot check.